lib/ingreedy/dictionary_collection.rb

```diff
@@ -8,7 +8,7 @@ def initialize
     end
 
     def []=(locale, attributes)
-      @collection[locale] = Dictionary.new(**attributes)
+      @collection[locale] = Dictionary.new(locale:, **attributes)
```

Comment:
To support versions of Ruby earlier than 3.1.

```suggestion
      @collection[locale] = Dictionary.new(**attributes, locale: locale)
```

Additionally, `locale` should proceed `**attributes` in order for the value to take priority over a `locale` value that might have erroneously been provided inside `attributes`

```ruby
irb(main):001> attributes = {foo: "bar", locale: "bad"}
=> {:foo=>"bar", :locale=>"bad"}
irb(main):002* def test(**values)
irb(main):003*   values[:locale]
irb(main):004> end
=> :test
irb(main):005> test(locale: "good", **attributes)  
=> "bad"
irb(main):006> test(**attributes, locale: "good")  
=> "good"
```

Reply:
> locale should come after **attributes
> ```ruby
>  irb(main):005> test(locale: "good", **attributes)  
>  => "bad"
>  irb(main):006> test(**attributes, locale: "good")  
>  => "good"
>  ```

Oh. I don't know this feature. I'll fix this. Thanks